src/rules/ig.toml

```diff
@@ -0,0 +1,23 @@
+min_trimmed_length = 3
+min_word_count = 3
+max_word_count = 14
+min_characters = 0
+may_end_with_colon = false
+quote_start_with_letter = true
+needs_punctuation_end = false
+needs_letter_start = true
+needs_uppercase_start = false 
```

Comment:
Looking at random Wikipedia articles it seems to me that the correct grammar would be with an uppercase letter at the start of a sentence. I might be wrong, but something you might want to look into :)

Reply:
You may be right. many of the extracted sentences did start with an uppercase letter

Reply:
If I had to guess, without knowing the language, I'd have the following input:

* The `|` within`Éʻ|c` might not be what you want?
* `\\.` within `\\.pqYð` should already be covered with `;‚'–\\. \"`
* `\”` doesn't need the `\` before, this leads to the one of the parse errors
* Once that is fixed, there is another error complaining about what you just posted while I was writing this. Happy you fixed it :)
* `;` and `,` might also be covered twice (didn't cross-check the actual code points though)

Reply:
There are some other duplications as well, such as `LER`. Might be worth cross-checking those and simplifying. Generally I would say, while it's still quite hard to read, it's still better than the disallow list. Nice work so far!

Reply:
It not being included might be due to the 3 sentences per article requirement and it just picked 3 other sentences from that article? You could pass `--no_check` at least for the validation (but that won't be used for the extract, as the 3 sentences per article rule is a strict limitation). With the `--no_check` flag it basically uses all sentences.

Reply:
Oh no, it actually doesn't match the `w` in that sentence as it's not allowed.

Reply:
i fixed the `w` error. it still did not show.

Reply:
Then it might also be the 3 sentences per article limit. I think for verification of the rules it would make sense to use `--no_check` flag when running the extractor.

Reply:
you're right. 
everything works well now.

Reply:
Sure, I'm not really concerned about performance here. I'd be fine with this approach. Please report back once the final review is done and I will take it from there. Thanks!